bundle/manifests/splunk-otel-collector-operator.clusterserviceversion.yaml

```diff
@@ -499,7 +504,7 @@ spec:
   provider:
     name: Splunk Inc
     url: github.com/signalfx/splunk-otel-collector-operator
-  version: 0.0.4
+  version: 0.0.1
```

Comment:
was this version change expected?

Reply:
It is, I hit the revert button twice by accident. The fix is already pushed 👍